composer.json

```diff
@@ -17,7 +17,7 @@
     "php": ">=5.6",
     "ext-json": "*",
     "ext-openssl": "*",
-    "google/apiclient": "^2.0",
+    "google/apiclient": "^2.9.1",
```

Comment:
@benbowler changing the version manually in the composer.json file is not enough because in this case the composer.lock file remains the same. We need to update the dependency version correctly which will mean that the lock file should be updated as well. 

Just run `composer remove google/apiclient` to remove the existing version and run `composer require google/apiclient` to install the latest one. It will correctly update composer.json and composer.lock files.

Reply:
Good point, I failed to commit the `composer.lock`. Pushed up now.